lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -268,16 +273,28 @@ def self.run(params)
 
         Options.strict_mode(params[:strict])
 
-        # if app found or successfully created
-        if self.get_or_create_app(params)
+        release = nil
+        case self.get_or_create_app(params)
+        when :created_app
+          each_retry = Proc.new do |exception, try, elapsed_time, next_interval|
+            UI.message("Release upload failed (#{exception.message}), retrying. Attempt# #{try}. #{next_interval} seconds until the next try.")
+          end
+          Retriable.retriable(tries: 60, on_retry: each_retry) do
```

Comment:
This seems to add a new dependency, but it's not added to Gemfile. Retriable is also not a direct dependency in fastlane.